.github/actions/install-briefcase/action.yml

```diff
@@ -0,0 +1,101 @@
+name: Install Briefcase
+description: Installs a version of Briefcase appropriate for the context.
```

Comment:
Is there any particular reason for the `actions/install-briefcase/action.yml` structure? Could this be simplified to `actions/install-briefcase.yml`?

Reply:
This is a requirement [defined](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions) by GitHub for composite actions.

Reply:
TIL.